packages/node_modules/samples/browser-meetings/app.js

```diff
@@ -372,3 +372,148 @@ function sendDtmfTones() {
+
+    buttons.forEach((buttonElement) => {
+      // eslint-disable-next-line no-param-reassign
+      buttonElement.value = id;
+    });
+    statusElemLabel.innerHTML = `audio status: ${statusObj.audioStatus} 
+    video status: ${statusObj.videoStatus}`;
+
+    idElemLable.innerHTML = id;
+    particpantNameLabel.innerHTML = name;
+  };
+
+  if (meeting) {
+    meeting.invite({emailAddress: emailVal}).then((res) => {
+      console.log(res.body.locus.participants);
+      const particpantsArr = res.body.locus.participants;
+      let mostRecentParticipantId;
+      let mostRecentParticpantName;
+      let mostRecentParticpantStatusObj;
+
+      particpantsArr.forEach((obj) => {
+        if (obj.person.email === emailVal ||
+          obj.person.primaryDisplayString === emailVal) {
+          mostRecentParticipantId = obj.id;
+          mostRecentParticpantName = obj.person.name ||
+          obj.person.primaryDisplayString;
+          mostRecentParticpantStatusObj = obj.status;
+        }
+      });
+
+      addParticipantsToTheDom(mostRecentParticipantId,
+        mostRecentParticpantName,
+        mostRecentParticpantStatusObj);
+    }).catch((err) => {
+      console.log('unable to invite participant', err);
+    });
+  }
+}
+
+function admitMember(admitButton) {
+  const meeting = getCurrentMeeting();
+  const particpantID = admitButton.value;
+
+  if (meeting) {
+    meeting.admit([particpantID]).then((res) => {
+      console.log(res, 'participant has been admitted');
+    }).catch((err) => {
+      console.log('unable to admit participant', err);
+    });
+  }
+}
+
+function removeMember(removeButton) {
+  const meeting = getCurrentMeeting();
+
+  const particpantID = removeButton.value;
+
+  if (meeting) {
+    meeting.remove(particpantID).then((res) => {
+      console.log(res, 'participant has been removed');
+    }).catch((err) => {
+      console.log('unable to remove participant', err);
+    });
+  }
+}
+
+function muteMember(muteButton) {
+  const meeting = getCurrentMeeting();
+  const text = muteButton.innerText;
+  const mute = text === 'Mute';
+  const particpantID = muteButton.value;
+
+  if (meeting) {
+    meeting.mute(particpantID, mute).then((res) => {
+      console.log(res, `participant is ${text}`);
+      if (mute) {
+        // eslint-disable-next-line no-param-reassign
+        muteButton.innerText = 'Unmute';
+      }
+      else {
+        // eslint-disable-next-line no-param-reassign
+        muteButton.innerText = 'Mute';
+      }
+    }).catch((err) => {
+      console.log('error', err);
+    });
+  }
+}
+
+function transferHostToMember(transferButton) {
+  const meeting = getCurrentMeeting();
+  const particpantID = transferButton.value;
+
+  if (meeting) {
+    meeting.transfer(particpantID).then((res) => {
+      console.log(res, `succesful tranfer to ${particpantID}`);
+    }).catch((err) => {
+      console.log('error', err);
+    });
+  }
+}
+
+function viewParticipants() {
```

Comment:
This button action should be mapped to injecting data into the above table in which each participant has `admit`, `remove`, `mute` and `transfer ownership` buttons. It appears that all the data needed is there, but the injected location needs to be updated.